packages/screens/CairoPOC/CairoPOCScreen.tsx

```diff
@@ -0,0 +1,163 @@
+import { useAccount, useContract } from "@starknet-react/core";
+import { useCallback, useEffect, useState } from "react";
+import { View } from "react-native";
+import { type Abi, shortString } from "starknet";
```

Comment:
Do we really need `type` usage ?
```suggestion
import { Abi, shortString } from "starknet";
```